app/assets/stylesheets/custom.scss

```diff
@@ -5,6 +5,12 @@
 
 $gray-medium-light: #eaeaea;
 
+@mixin box_sizing {
+  -moz-box-sizing:    border-box;
```

Comment:
[FYI]
You can try mixin here https://sass-lang.com/guide
It's good for adding prefixes